gravity_upgrade.md

```diff
@@ -72,25 +73,126 @@ A manual upgrade operation starts with an operation plan which is a tree of acti
 
 ## Gravity Upgrade Status, Logs, and more
 
+In order to monitor the cluster, it's useful to go through logs and status messages provided by the platform.
+
+Below there's a list of useful commands/resources to use during debugging.
+
+The list is divided based on when the specific command will be more likely used, like during upgrade or at runtime.
+
+### DURING INSTALL / UPGRADE
```

Comment:
Also need to add commands related to upgrade agents. How to check the agent is running (gravity-agent.service), how to check its logs, how to redeploy, etc. Plus, all of those things that @aelkugia mentioned below (resuming plan, checking CLI history, etc.).